### PR TITLE
Responsive Styles

### PR DESCRIPTION
**Pull request by fdarosa2663**
added responsive_styles.h and made change to LB button to make control less cumbersome for users

### CODE REVIEW

File: ProffieOS.ino [hunk 1]
```diff
@@ -1,1666 +1,1670 @@
...
-
-#include "common/clock_control.h"
-
-void loop() {
-#ifdef MTP_RX_ENDPOINT
-  mtpd.loop();
-#endif
-  Looper::DoLoop();
-}
+/*
```
Comment (profezzorn) [inline ProffieOS.ino]: This diff shows every line in ProffieOS.ino has been changed somehow.
I don't know why that is, but I suspect that maybe line endings have been changed from NL to CRLF-NL
Whatever the reason, it makes it very hard to see what the actual changes are.

**Comment by fdarosa2663** [comment]
The only change to the .ino was to include the responsive_styles.h below transitions. I made the changes in Arduino 1.8.9 would that cause?  

----added----
// responsive styles
#include "styles/responsive_styles.h"
------------------------

should I replace with original .ino and make same change again?

File: ProffieOS.ino (hunk 1 above)
Comment (fdarosa2663) [reply ProffieOS.ino]: The only change to the .ino was to include the responsive_styles.h below transitions. I made the changes in Arduino 1.8.9 would that cause?

----added----
// responsive styles
#include "styles/responsive_styles.h"
should I replace with original .ino and make same change again?

Comment (profezzorn) [reply ProffieOS.ino]: You need to fix the difff though, or I can't merge the change.

**Comment by fdarosa2663** [comment]
will redo to fix differences

**Pull request closed without merging**